### PR TITLE
sentry

### PR DESCRIPTION
**Pull request by maggieprice**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const server = require("./data/server.js");
 const PORT = process.env.PORT || 5000;
+const Sentry = require('@sentry/node');
+Sentry.init({ dsn: 'https://c0e164d7a07b4f54854d29fc3c62f50f@sentry.io/5167996' });
```
Comment (israelgonzalezb) [inline index.js]: To clarify, it looks like this key is not secret. We can choose to move it to .env but it is not critical. Sentry gives us a public key and a secret key. This is the public key.

https://docs.sentry.io/error-reporting/configuration/?platform=node#dsn

File: users/users-model.js [hunk 2]
```diff
@@ -137,4 +137,6 @@ function edituserProfile(Userid, body){
   .returning('id')
   .where({id:Userid})
   .update(body, "id")
-}
\ No newline at end of file
+}
+
+myUndefinedFunction();
```
Comment (israelgonzalezb) [inline users/users-model.js]: What is this?

Comment (maggieprice) [reply users/users-model.js]: the sentry documentation called for this anonymous function to be placed in the code so that it can throw an exception

**Comment by israelgonzalezb** [comment]
I changed the base branch from master to development.

**Pull request merged**